js/ui/sortable.js

```diff
@@ -236,9 +238,16 @@ const Sortable = Draggable.inherit({
 
         if($scrollable) {
             const offset = $scrollable.offset();
-            const validY = offset.top + $scrollable.height() >= event.pageY && offset.top <= event.pageY;
-            const validX = offset.left + $scrollable.width() >= event.pageX && offset.left <= event.pageX;
-
+            // use getComputedStyle, because vertical scrollbar reduces content width
+            const style = window.getComputedStyle($scrollable[0]);
+            const paddingLeft = parseFloat(style.paddingLeft) || 0;
+            const paddingTop = parseFloat(style.paddingTop) || 0;
+            const width = parseFloat(style.width) || 0;
+            const height = parseFloat(style.height) || 0;
+            const startX = offset.left + paddingLeft;
+            const startY = offset.top + paddingTop;
```

Comment:
Can you separate this into a separate method?

For example getScrollableBoundary, which returns {top, bottom, left, right}